addons/mail/static/src/core/web/notification_item.dark.scss

```diff
@@ -0,0 +1,9 @@
+
+.o-mail-NotificationItem {
```

Comment:
comment not on the right line, but `/mail/static/src/img/smiley/mailfailure.jpg` doesn't work in dark theme because it has white background built-in

Reply:
Yes, this is a specific problem with the image using a white background instead of alpha like SMS failure does.
We should probably do a PR for this specific change